ports/rp2/cyw43_configport.h

```diff
@@ -26,34 +26,27 @@
 #ifndef MICROPY_INCLUDED_RP2_CYW43_CONFIGPORT_H
 #define MICROPY_INCLUDED_RP2_CYW43_CONFIGPORT_H
 
-// The board-level config will be included here, so it can set some CYW43 values.
-#include <stdio.h>
-#include "py/mpconfig.h"
-#include "py/mperrno.h"
-#include "py/mphal.h"
-#include "py/runtime.h"
-#include "extmod/modnetwork.h"
-#include "lwip/apps/mdns.h"
-#include "pendsv.h"
+#include "extmod/cyw43_config_common.h"
 
 #define CYW43_INCLUDE_LEGACY_F1_OVERFLOW_WORKAROUND_VARIABLES (1)
 #define CYW43_WIFI_NVRAM_INCLUDE_FILE   "wifi_nvram_43439.h"
-#define CYW43_IOCTL_TIMEOUT_US          (1000000)
 #define CYW43_SLEEP_MAX                 (10)
```

Comment:
I haven't moved this macro into the common header because it's `50` in the mimxrt & stm32 ports, and `10` here in rp2. I'm unsure if this difference is necessary or a historical accident.

Reply:
It just controls how often the the cyw43 poll gets scheduled via pendsv (at least in stm32, mimxrt, alif). I'd say on a slower chip maybe 50 is better.

Reply:
Oh! Perhaps just have the default at 50 and in RP2 `undef` and redefine it.

Reply:
Right! What I'm unsure is whether it was set to 10 on rp2 to fix some problem (or improve performance), or whether it's simply an accident that it's been set to 10 here. I haven't done any tests to compare 50 vs 10 on rp2.

Reply:
> Oh! Perhaps just have the default at 50 and in RP2 `undef` and redefine it.

Good idea. I've gone even further and removed it from all the ports except rp2, as [50 is the default in the driver repo](https://github.com/georgerobotics/cyw43-driver/blob/main/src/cyw43_config.h#L126). I suspect it could also be set to 50 in the rp2 port without any impact (maybe even lower sleep consumption), but haven't tested this.